src/health-check.js

```diff
@@ -2,26 +2,21 @@
 const ioOperations = require('./io-operations.js')
 
 exports.runCheck = function() {
-  return new Promise((resolve, reject) => {
-    let firstThingRead
-    ioOperations.readManifest()
+  return ioOperations
```

Comment:
I was a n00b when I first wrote this. Returning promises from inside of a .then() allows you to not have to do lots of .catch()es or make `new Promise()`